index.js

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright (c) 2019 Digital Bazaar, Inc. All rights reserved.
+*/
+'use strict';
+
+const program = require('commander');
+const {api, key, testApi} = require('./lib');
+
+// debuggers
+// const loaderDebug = require('debug')('vc:loader');
+
+// function inc(value, total) {
+//   return total + 1;
+// }
+
+/*
+if(process.env.JSONLD_LOADER_MAP) {
+  const map = {};
+  const cache = {};
+  const oldLoader = vc.documentLoader;
+  process.env.JSONLD_LOADER_MAP
+    .split(',')
+    .map(v => v.split('='))
+    .forEach(v => map[v[0]] = v[1]);
+  loaderDebug('map: %O', map);
+  vc.documentLoader = async (url) => {
+    if(url in cache) {
+      // already loaded
+      loaderDebug('from cache: %s', url);
+    } else if(url in map) {
+      // load from file
+      loaderDebug('from file: %s', url);
+      cache[url] = {
+        contextUrl: null,
+        documentUrl: url,
+        document: JSON.parse(await fs.readFile(map[url]))
+      };
+    } else {
+      // load with default loader
+      loaderDebug('default: %s', url);
+      cache[url] = await oldLoader(url);
+    }
+    return cache[url];
+  };
+}
+*/
+
+// program
+//   .version(require('../package.json').version)
+//   .option('-v, --verbose', 'Increase verbosity', inc, 0);
+
+program
+  .command('generate-key')
+  .description('Generate an Ed25519 key pair')
+  .option('-t, --token <token>', 'GitHub personal access token.')
```

Comment:
We should be aligning w/ openssl here... -t means "key type" there, people are used to it. Suggest '-t', and '--key-type'.

Make the Github stuff be '-g', and '--github-token'.